qiskit/schemas/backend_configuration_schema.json

```diff
@@ -1,6 +1,6 @@
 {
     "$schema": "http://json-schema.org/draft-04/schema#",
-    "id": "http://www.qiskit.org/schemas/backend_config_schema.json",
```

Comment:
Looks like replacement mistake to schema files in this directory.

Reply:
Yes, already taken care of via 4f8ac24 🙂